sql/database.sql

```diff
@@ -5003,7 +5004,6 @@ CREATE TABLE `patient_data` (
   `state` varchar(255) NOT NULL default '',
   `country_code` varchar(255) NOT NULL default '',
   `drivers_license` varchar(255) NOT NULL default '',
-  `ss` varchar(255) NOT NULL default '',
```

Comment:
Will break code base if removed

Reply:
Social Security Numbers break the code?  Where?  Shouldn't that be a Localization thing in any case?
No provision in the Medicare manual for gathering SSN...but I will see what is what.

Reply:
Yeah, I knew about the dupecheck.  That was one of the things that always made using the merge patient such a pain in the ass.  People would use the ssn field for all kinds of stuff, then have to open up each patient, edit both, run the merge again etc...